azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -329,13 +299,13 @@ private void onTriggerExpire(Trigger t) throws TriggerManagerException {
       List<TriggerAction> expireActions = t.getExpireActions();
       for (TriggerAction action : expireActions) {
         try {
-          logger.info("Doing expire actions");
+          logger.info("Doing expire actions for "+ action.getDescription() + " for " + t);
```

Comment:
lots of places in this code uses this pattern. created an independent refactor request to track.